modules/@apostrophecms/asset/lib/webpack/media-to-container-queries-loader.js

```diff
@@ -0,0 +1,87 @@
+const postcss = require('postcss');
+
+module.exports = function (source) {
+  const schema = {
+    title: 'Media to Container Queries Loader options',
+    type: 'object',
+    properties: {
+      debug: {
+        type: 'boolean'
+      },
+      transform: {
+        anyOf: [
+          { type: 'null' },
+          { instanceof: 'Function' }
+        ]
+      }
+    }
+  };
+  const options = this.getOptions(schema);
+
+  const mediaQueryRegex = /@media\s*(all|print|screen(?: and | or )?)?([^{]+)[^{]*{([\s\S]*?})\s*(\\n)*}/g;
+
+  const convertToContainerQuery = (mediaFeature, content) => {
```

Comment:
I used to be keenly against it (@ValJed would remember) but now, I think that I like having the functions definition placed after their call. I find it much more easier to read in the file (main function -> call to another function -> function definition).

I.e: you could move this function at the end, so we read the code execution in the execution order.
**(this is very optional)**.

Reply:
I really have a hard time with hoisting

Reply:
and the entire file is 80+ lines of code